src/utils/breakpoints.js

```diff
@@ -21,8 +22,12 @@ export const BreakpointScopes = {
 
 export const BreakpointAttributes = {
   [BreakpointScopes.default]: {
+    [BreakpointName.xlarge]: {
+      minWidth: 1920,
+      contentWidth: 1536,
+    },
     [BreakpointName.large]: {
-      minWidth: 1069,
+      minWidth: 1251,
```

Comment:
This change was to equal the values with the [_breakpoints.scss](https://github.com/apple/swift-docc-render/blob/0fa8f75737aa071cc9fa0bfd2126c5d7c152c7c3/src/styles/core/_breakpoints.scss) file